beacon_node/network/src/router.rs

```diff
@@ -664,20 +648,19 @@ impl<T: BeaconChainTypes> Router<T> {
             AppRequestId::Sync(sync_id) => match sync_id {
                 id @ SyncRequestId::SingleBlock { .. } => id,
                 other => {
-                    crit!(self.log, "BlocksByRoot response on incorrect request"; "request" => ?other);
+                    crit!( request = ?other, "BlocksByRoot response on incorrect request");
```

Comment:
```suggestion
                    crit!(request = ?other, "BlocksByRoot response on incorrect request");
```

More `crit`s in this file also have this extra space